### PR TITLE
test with simple source filter rather than Switch.pm

### PR DESCRIPTION
**Pull request by haarg**
Switch is a far too complex source filter and is best avoided as a prereq. Replace it with a simple source filter included with the dist.

### CODE REVIEW

File: t/switch.cgi [hunk 1]
```diff
@@ -2,8 +2,7 @@
...
 use strict;
 use warnings;
-use Switch;
+use lib 't';
+use SimpleFilter;
 
-switch (42) {
-    case 42 { print "switch works\n" }
-}
+print "filter works\n";
```
Comment (miyagawa) [inline t/switch.cgi:8]: did you mean `tnirp` here?

Comment (haarg) [reply t/switch.cgi:8]: Yes.

**Comment by miyagawa** [comment]
LGTM

**Comment by rkitover** [comment]
@miyagawa Want me to upload this?

**Comment by rkitover** [comment]
Uploaded 0.26.

**Pull request merged**